fw/t/unit/test_http_parser_common.h

```diff
@@ -649,19 +648,25 @@ do {									\
 	    FUZZ_REQ_H2, CHUNK_ON)
 #define EXPECT_BLOCK_REQ_H2_CHUNK_OFF(str)				\
 	__EXPECT_BLOCK_REQ(str, FUZZ_REQ_H2, CHUNK_OFF)
+#define EXPECT_BLOCK_REQ_H2_HPACK(H2_FRAMES_DEF_BLOCK)			\
+	INIT_FRAMES();							\
+	SET_FRAMES_BUF(frames_buf);					\
+	H2_FRAMES_DEF_BLOCK;						\
+	RESET_FRAMES_BUF();						\
+	TRY_PARSE_EXPECT_BLOCK(FUZZ_REQ_H2, CHUNK_ON)
```

Comment:
Please use `do { .. } while (0)` for multi-line defines here and for all the code.

Also please empty lines between multi-line macro definitions

Reply:
In the case of FOR_REQ_H2_HPACK and others like it, this is not possible. Because after FOR_REQ_H2_HPACK you can add a check block, which is actually the body of the while loop from TRY_PARSE_EXPECT_PASS.
In the current PR I can use `do { .. } while (0)` for the EXPECT_BLOCK_REQ_H2_HPACK case because TRY_PARSE_EXPECT_BLOCK cannot have a check block like TRY_PARSE_EXPECT_PASS. But in some next PR, EXPECT_BLOCK_REQ_H2_HPACK already has the ability to use a check block similar to FOR_REQ_H2_HPACK.